#!/bin/sh /usr/share/dpatch/dpatch-run
## 37-magic-add-syslinux.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: FIXME

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/images file-4.20/magic/Magdir/images
--- file-4.20.orig/magic/Magdir/images	2007-03-27 17:58:55.000000000 +0000
+++ file-4.20/magic/Magdir/images	2007-03-27 17:59:58.000000000 +0000
@@ -527,3 +527,9 @@
 # Ulead Photo Explorer5 (.pe5)
 # Source: http://www.jisyo.com/cgibin/view.cgi?EXT=pe5 (Japanese)
 0	string	IIO2H		Ulead Photo Explorer5
+
+# The boot loaders syslinux and isolinux use a RLE based image format
+# called SLL16 to store splash screens.
+0	lelong	0x1413f33d	Syslinux SLL16 image data,
+>4	leshort	>0		%hd x
+>6	leshort	>0		%hd
